packages/front-end/components/Archetype/ArchetypeList.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { FC, useState } from "react";
+import { ArchetypeInterface } from "back-end/types/archetype";
+import Link from "next/link";
+import usePermissionsUtil from "@/hooks/usePermissionsUtils";
+import { useAuth } from "@/services/auth";
+import track from "@/services/track";
+import { GBAddCircle } from "@/components/Icons";
+import Tooltip from "@/components/Tooltip/Tooltip";
+import Code from "@/components/SyntaxHighlighting/Code";
+import styles from "@/components/Archetype/ArchetypeResults.module.scss";
+import MoreMenu from "@/components/Dropdown/MoreMenu";
+import DeleteButton from "@/components/DeleteButton/DeleteButton";
+import ArchetypeAttributesModal from "@/components/Archetype/ArchetypeAttributesModal";
+import { useDefinitions } from "@/services/DefinitionsContext";
+
+export const ArchetypeList: FC<{
+  archetypes: ArchetypeInterface[];
+  archetypeErrors: Error | undefined;
+  mutate: () => void;
+}> = ({ archetypes, archetypeErrors, mutate }) => {
+  const [
+    editArchetype,
+    setEditArchetype,
+  ] = useState<Partial<ArchetypeInterface> | null>(null);
+  const permissionsUtil = usePermissionsUtil();
+  const { project, getProjectById } = useDefinitions();
+
+  const canCreateGlobal = permissionsUtil.canCreateArchetype({
+    projects: [project],
+  });
+
+  const { apiCall } = useAuth();
+
+  if (archetypeErrors) {
+    return (
+      <div className="alert alert-danger">
+        An error occurred fetching the lists of archetypes.
+      </div>
+    );
+  }
+
+  return (
+    <>
+      <div className="row mb-3">
+        <div className="col">
+          <h1>Archetypes</h1>
+        </div>
+        {canCreateGlobal && (
+          <div className="col-auto">
+            <button
```

Comment:
use the new Radix button class with no "+" icon for consistency

Reply:
Done, though I think the new button is way too big